server/etcdserver/server.go

```diff
@@ -1001,6 +1001,7 @@ func (s *EtcdServer) applySnapshot(ep *etcdProgress, toApply *toApply) {
 	// wait for raftNode to persist snapshot onto the disk
 	<-toApply.notifyc
 
+	// gofail: var beforeOpenSnapshotBackend struct{}
```

Comment:
When I see the name, the immediately feeling is that it's used on bootstrap when opening a snapshot backend.

```suggestion
	// gofail: var beforeOpenSnapshotBackendInApply struct{}
```

Reply:
Should it be prefix or suffix based? Failpoints for raft have a nice `raft` prefix like `raftBeforeLeaderSend`. However it's less consistent for server/apply. For example `beforeCommit`, `beforeApplyOneConfigChange`.

Food for thought. For now I will pick your proposal as I cannot think of anything better.

Reply:
Done